frontend/src/components/alert/alert-styles.ts

```diff
@@ -0,0 +1,62 @@
+import styled from 'styled-components';
+import { AlertBoxProp } from '../../interfaces/styles';
+import { ButtonProps } from '../../interfaces/types';
+import { theme } from '../../theme';
+
+export const AlertBox = styled.div<AlertBoxProp>`
+  position: absolute;
+  top: 20px;
+  right: 20px;
+  display: flex;
+  width: 500px;
+  height: 100px;
+  border-radius: 10px;
+  box-shadow: 0px 0px 8px 0px ${theme.colors.shadow};
+  color: ${({ type }) => {
+    if (type === 'error') {
+      return theme.colors.danger;
```

Comment:
you have error, success string in multiple places..I would put these in some consts and create an enum